include/openthread/instance.h

```diff
@@ -45,6 +45,14 @@
 extern "C" {
 #endif
 
+/**
+ * The OpenThread API monotonic version number.
+ *
+ * The number should be increased everytime the OpenThread API changes.
```

Comment:
how can we make sure it's not violated?

Reply:
what about adding a GitHub Action check or git hook to verify that this version number is increased if the PR includes a change in `include/openthread`?